### ISSUE TITLE
Help menu items point to generic Electron URLs

### ISSUE THREAD

**Issue by johnblommers** [michalczaplinski/hypermark#5]
The Hypermark Help menu has four items and all point to generic Electron web pages:

- Learn More -> https://electronjs.org/
- Documentation -> https://github.com/electron/electron/tree/master/docs#readme
- Community Discussions -> https://discuss.atom.io/c/electron
- Search Issues -> https://github.com/electron/electron/issues

Surely all of the links should be to URLs specific to Hypermark?

**Comment by michalczaplinski** [comment]
@johnblommers yes, those should definitely point to hypermark-specific documentation once it actually exists! :)

If there's some documentation or link that you'd like to see in particular, you've very welcome to open an issue or send a PR at https://github.com/michalczaplinski/hypermark-website and I ll take a look at it ASAP! 🙂
